lib/doctor/DiagnosticInfoBuilder.ts

```diff
@@ -36,6 +39,7 @@ export interface DiagnosticInfo extends FilesInfo {
   path?: string;
   versions: { [hubspotCli]: string; node: string; npm: string | null };
   config: string | null;
+  configSettings: Record<string, unknown>;
```

Comment:
I think we're staying away from the `Record` pattern in favor of a plain object.

Reply:
Ah good catch. I fixed this in one place and forgot to do it here too